src/WebJobs.Script/Diagnostics/DiagnosticEventConstants.cs

```diff
@@ -28,5 +28,8 @@ internal static class DiagnosticEventConstants
 
         public const string MissingFunctionsWorkerRuntimeErrorCode = "AZFD0011";
         public const string MissingFunctionsWorkerRuntimeHelpLink = "https://go.microsoft.com/fwlink/?linkid=2257963";
+
+        public const string NonHISSecretLoaded = "AZFD0012";
+        public const string NonHISSecretLoadedHelpLink = "https://aka.ms/functions-non-his-secrets";
```

Comment:
This is link that I generated that points to the internal page we have as part of the Project Standard Secrets work. Initially we'll be using this feature for first party internal customers. However, given this is a feature that the public could also use if they wish, need to think about what this should point to.

Reply:
What I'll do is write up an initial wiki help page on https://github.com/Azure/azure-functions and update the aka link to point to that. I don't think finalizing this needs to block the PR since that link can be updated independently.

Reply:
We have a pattern for diagnostic events and they're all setup here: https://learn.microsoft.com/en-us/azure/azure-functions/errors-diagnostics/diagnostic-events/azfd0001

Let me know if you need any help with this. The links also feed into reports used to make documentation improvements.

Reply:
I'll create a separate PR adding a new page here as you suggest. I'll do this separate from this PR, and update the aka link when done.
![image](https://github.com/Azure/azure-functions-host/assets/1056548/dbff93be-e9bc-43ed-9809-4d4c7bbeb1d4)

Reply:
Doc PR is out here: https://github.com/MicrosoftDocs/azure-docs/pull/120308